node/errors.js

```diff
@@ -628,6 +688,7 @@ module.exports.classify = function classify(err) {
         case 'tchannel-thrift-handler.stringify-error.body-failed':
         case 'tchannel-thrift-handler.stringify-error.head-failed':
         case 'tchannel.argstream.unknown-frame-handling-state':
+        case 'tchannel.connection.unknown-reset':
```

Comment:
I don't think we can say that with confidence, all this means is "someone forgot to pass an error to conn.resetAll"